### PR TITLE
Created new Cask file for Sony Ericsson Bridge

### PR DESCRIPTION
**Pull request by welly**
Useful application for Sony Xperia Android users

### CODE REVIEW

File: Casks/sony-ericsson-bridge.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'sony-ericsson-bridge' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www-support-downloads.sonymobile.com/Software%20Downloads/Bridge%20for%20Mac/SonyBridgeForMac_web.dmg'
+  name 'Sony Bridge for Mac'
```
Comment (vitorgalvao) [inline Casks/sony-ericsson-bridge.rb]: Shouldn’t it be `Sony Ericsson Bridge for Mac`?

**Comment by welly** [comment]
Fixed the name, apologies!

**Comment by vitorgalvao** [comment]
No worries, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by welly** [comment]
Squashed commits!

**Comment by vitorgalvao** [comment]
Actually, there are more commits, now. Squashing is difficult to grasp the first times we try it. If you need help or want me to do it, let me know.

**Comment by welly** [comment]
Ah, my apologies. This is fairly (read that as very) new to me! Any advice is welcome!

**Comment by vitorgalvao** [comment]
There was something weird going on with repeated commits that made the process a bit different than usual, so I went ahead and fixed it in https://github.com/caskroom/homebrew-cask/pull/9347. Thank you for the contribution, it is still included (and still credited to you), with the appropriate modifications.

**Comment by vitorgalvao** [comment]
One thing I’m noticing, though. Is this app paid? It seems not to be, in which case it should be `:gratis`, not `:commercial`.

**Comment by welly** [comment]
Oh, I assumed that it was commercial as it isn’t open source. Apologies for this comedy of errors on my part!

**Comment by vitorgalvao** [comment]
No worries. [Here’s how we classify licenses](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#valid-licenses).

**Comment by welly** [comment]
Ok, gotcha. Shall I update it with the revised license? Before I start making a billion commit messages, how should I revise this?

**Comment by vitorgalvao** [comment]
It’s already fixed, so it’s fine, you don’t have to do anything else. Thank you again for the contribution, and don’t be discouraged, it was definitely a good: the mistakes were small kinks.

**Comment by welly** [comment]
Great, thanks! I shall keep contributing and will learn a bit from this. Cheers!

**Pull request closed without merging**